src/parser.js

```diff
@@ -670,6 +670,12 @@ function mapClassPrivateVariables(classNode, meta) {
   );
 }
 
+function mapInnerClasses(classNode, meta) {
+  return classNode.body.expressions
+    .filter(classExpression => classExpression.constructor.name === 'Class')
+    .map(innerClass => b.expressionStatement(mapExpression(innerClass, meta)));
```

Comment:
Do you need the wrapping `expressionStatement`? I didn't think class declarations were expressions, though they were their own top-level thing.